### PR TITLE
Allow disabling empty() call check

### PR DESCRIPTION
**Pull request by emodric**

### CODE REVIEW

File: rules.neon [hunk 1]
```diff
@@ -12,7 +13,6 @@ rules:
 	- PHPStan\Rules\BooleansInConditions\BooleanInElseIfConditionRule
 	- PHPStan\Rules\BooleansInConditions\BooleanInIfConditionRule
 	- PHPStan\Rules\BooleansInConditions\BooleanInTernaryOperatorRule
-	- PHPStan\Rules\DisallowedConstructs\DisallowedEmptyRule
```
Comment (emodric) [inline rules.neon]: I'm not sure if this should've been removed or not.

**Comment by ondrejmirtes** [comment]
Hi,
if you don't want the `empty` check, you can just use `ignoreErrors`.

Your proposed solution is not very systematic and should be solved in the future by general system for excluding rules. It's not possible to do it right now in a better way.

**Comment by emodric** [comment]
@ondrejmirtes You use 4 `check*` parameters here already, I don't see a reason why not allow one more. Yes, I can use `ignoreErrors`, but it has a tendency to break if you change the error message, while this parameter feels a bit more semantic.

Anyways, you're the boss :)

**Comment by ondrejmirtes** [comment]
Those `check*` parameters influence some details inside the rules but they do not disable rules entirely 😊

**Comment by ondrejmirtes** [comment]
And I'm aware that using `ignoreErrors` isn't ideal, but I already said that:

> should be solved in the future by general system for excluding rules

**Pull request closed without merging**